pure_pursuit_node:
  ros__parameters:
    lookahead_distance: 0.9
    max_lookahead: 1.1
    min_lookahead: 0.7
    curvature_gain: 0.5
    #Left
    lane1_file_path: "/config/icra_race_2_waypoints.csv"
    #center
    lane2_file_path: "/config/icra_race_2_waypoints.csv"
    #right
    lane3_file_path: "/config/icra_race_2_waypoints.csv"
    max_speed: 7.0
    min_speed: 2.0
    max_steering_angle: 0.7853981633974483  # np.pi / 4
    sim: True # set to False for real car 
    map_sim: True

pure_pursuit_node_d:
  ros__parameters:
    lookahead_distance: 0.9 #Not used if using adjustable lookahead
    max_lookahead: 1.1
    min_lookahead: 0.7
    curvature_gain: 0.5
    waypoint_file_path_in_package: "/config/icra_race_2_waypoints.csv"
    # waypoint_file_path_in_package: "/config/wp-2025-03-19-00-25-36.csv"
    max_speed: 7.0
    min_speed: 2.0
    max_steering_angle: 0.7853981633974483  # np.pi / 4
    sim: True # set to False for real car will handle the next 2 params odom_topic and drive_topic also automatically
    odom_topic: "/ego_racecar/odom" # "/opp_racecar/odom" The parameter is only used in sim mode as otherwise subscribe to the topic "pf/pose/odom" with sim False is programmed
    drive_topic: "/drive" # "/opp_drive" The parameter is only used in sim mode as otherwise publish to the topic "/drive" with sim False is programmed
    map_sim: True
    is_debug: False